airbyte-cdk/python/unit_tests/sources/streams/http/test_http.py

```diff
@@ -167,7 +167,7 @@ def max_retries(self):
     req.status_code = HTTPStatus.TOO_MANY_REQUESTS
     send_mock = mocker.patch.object(requests.Session, "send", return_value=req)
 
-    with pytest.raises(UserDefinedBackoffException):
+    with pytest.raises(UserDefinedBackoffException, match="\nRequest URL: https://test_base_url.com/\nResponse Code: 429"):
```

Comment:
Could you unit test the change you also made to `BaseBackoffException`?

Reply:
I added one test for `DefaultBackoffExeption` but will spend some time to add a new test `BaseBackoffException`